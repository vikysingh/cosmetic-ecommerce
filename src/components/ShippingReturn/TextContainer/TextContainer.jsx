import React from "react"

import { Typography } from "antd"

import styles from "./TextContainer.module.css"

import PropTypes from "prop-types"

const { Title, Paragraph } = Typography

export default function TextContainer({ title, children }) {
    return <div className={styles.TEXT_CONTAINER} >
        <Title level={4}> { title } </Title>
        <Paragraph>
            { children }
        </Paragraph>
    </div>
}

TextContainer.propTypes = {
    title: PropTypes.string,
    children: PropTypes.node
}