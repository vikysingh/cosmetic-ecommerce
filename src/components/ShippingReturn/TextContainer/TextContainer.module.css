.TEXT_CONTAINER {
    height: auto;
    margin-top: 30px;
    margin-bottom: 30px;
}

.TEXT_CONTAINER h4 {
    margin-bottom: 3px;
    text-transform: unset;
}

.TEXT_CONTAINER > p {
    width: 100%;
    opacity: 0.8;
}