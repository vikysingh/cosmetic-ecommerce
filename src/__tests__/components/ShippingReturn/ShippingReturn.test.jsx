import React from "React"
import { shallow } from "enzyme"

import TextContainer from "../../../components/ShippingReturn/TextContainer/TextContainer"

test('Should render TextContainer from ShippingReturn/TextContainer', () => {
    const wrapper = shallow(<TextContainer title="Title"> hello </TextContainer>)

    expect(wrapper).toMatchSnapshot()
})