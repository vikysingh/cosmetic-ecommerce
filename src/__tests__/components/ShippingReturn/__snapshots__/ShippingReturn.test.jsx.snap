// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render TextContainer from ShippingReturn/TextContainer 1`] = `
<div
  className="TEXT_CONTAINER"
>
  <Title
    level={4}
  >
     
    Title
     
  </Title>
  <Paragraph>
     hello 
  </Paragraph>
</div>
`;
